SPECS/gcc/gcc.spec

```diff
@@ -288,7 +496,18 @@ $tests_ok
 %{_lib64dir}/libgomp.so
 %{_lib64dir}/libgomp.spec
 
+%if %{build_cross}
+%files -n cross-gcc-common
+%license COPYING
+%endif
+
+%do_files aarch64-linux-gnu %{build_aarch64}
```

Comment:
This doesn't really make sense to me, why do we need to hard-code `aarch4-linux-gnu` here? Could you add a comment either here, or with `do_files` explaining the logic?

Reply:
I've added a comment explaining `do_files` and `do_exclude` - good idea. The macros are adding files for each cross-compilation architecture. Right now there's only one, so we could drop the macros but this way it's easier to add any other architectures in the future. I also wanted to keep it similar to Fedora's way of doing thing, since we often rely on their specs to update ours, so that future devs may have it slightly easier.